cmd/lite/main_test.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestFlags(t *testing.T) {
+	getConfigsFromCommandLine()
+	t.Log("Success (no panics for 'flag redefined')")
```

Comment:
I don't think this Log is necessary. You could use `recover` to catch any panics and fail the test with a useful message.

Reply:
Or, use tesitfy's `NotPanics` https://godoc.org/github.com/stretchr/testify/assert#NotPanics